cicd/pipelines/reducing-pipelines-resource-consumption.adoc

```diff
@@ -0,0 +1,27 @@
+[id="reducing-pipelines-resource-consumption"]
+= Reducing resource consumption of pipelines
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+:context: reducing-pipelines-resource-consumption
+
+toc::[]
+
+
+If you use clusters in multi-tenant environments you must control the resource (cpu and memory) consumption per namespace or resource in order to prevent one application from consuming more than it should and creating problems for other applications.
```

Comment:
Sentence sounds too big. Consider using punctuations and breaking into smaller sentences. I would suggest:
For clusters in multi-tenant environments, you must control the consumption of CPU and memory per namespace or resource. This helps prevent one application from consuming too much resource and creating problems for other applications.